critiquebrainz/db/review_test.py

```diff
@@ -211,6 +207,7 @@ def test_list_reviews(self):
         self.assertEqual(count, 1)
         self.assertEqual(len(reviews), 1)
 
+    @mock.patch('brainzutils.cache.get', Mock(return_value=None))
```

Comment:
this is a different syntax to the other uses of `mock.patch`. Can we use the other version, where you pass an argument into the method?